basilisk/tests/test_anthropic_engine.py

```
@@ -0,0 +1,890 @@
+            ChatMessage(role="user", content="User message"),
+        ]
+
+        result = engine._convert_messages(messages)
+
+        assert len(result) == 1
+        assert result[0] == {"role": "user", "content": "User message"}
+        assert "System message found in messages list" in caplog.text
+
+    def test_convert_messages_empty_content_skipped(self, sample_config):
+        """Test that messages with empty content are skipped."""
+        engine = AnthopicEngine(sample_config)
```

Ruff (0.11.9)

Undefined name `AnthopicEngine`

(F821)